iot-hub/Samples/device/PnpDeviceSamples/PnpConvention/PnpConvention.cs

```diff
@@ -112,11 +105,11 @@ public static string CreatePropertyPatch(string propertyName, object propertyVal
         /// All properties are read-write from a device's perspective.
         /// For a root-level property update, the patch is in the format: <c>{ "samplePropertyName": 20 }</c>
         /// </remarks>
-        /// <param name="propertyKeyValuePairs">The twin properties and values to update.</param>
+        /// <param name="propertyPairs">The twin properties and values to update.</param>
         /// <returns>A compact payload of the properties to update.</returns>
-        public static string CreatePropertyPatch(IDictionary<string, object> propertyKeyValuePairs)
+        public static string CreatePropertyPatch(IDictionary<string, object> propertyPairs)
         {
-            return JsonConvert.SerializeObject(propertyKeyValuePairs);
+            return JsonConvert.SerializeObject(propertyPairs);
```

Comment:
nit: another feedback was to return the `TwinCollection` directly, instead of the string. Is that something we investigated?

Reply:
It almost occurred to me at one point. I like the idea.